src/pages/HistoryPage/HistoryPage.jsx

```diff
@@ -0,0 +1,122 @@
+import { Navbar, Sidebar } from "../../components";
+import { useTheme, useVideo } from "../../context";
+import React from "react";
+import { MdPlaylistPlay, MdWatchLater, MdUnpublished } from "react-icons/md";
+import { FaThumbsUp, FaRegThumbsUp } from "react-icons/fa";
```

Comment:
import { MdPlaylistPlay, MdWatchLater, MdUnpublished } from "react-icons/md";
import { FaThumbsUp, FaRegThumbsUp } from "react-icons/fa";
see here you are importing like this what if you want import two more different icons from different path you will end up with wrting two more lines so instead you abstract it out